NEWS.md

```diff
@@ -5,6 +5,10 @@
 * `seq_linter()` additionally lints on `1:n()` (from dplyr) 
   and `1:.N` (from data.table)  (#1396, @IndrajeetPatil).
 
+## New features
+
+* `commented_code_linter()` now lints commented argument code as well (#386, @AshesITR).
```

Comment:
I don't find the description here very clear -- could you revisit? maybe a quick example is warranted?